ts/features/idpay/initiative/configuration/screens/IbanConfigurationLandingScreen.tsx

```diff
@@ -27,12 +35,44 @@ const styles = StyleSheet.create({
 
 const IbanConfigurationLanding = () => {
   const configurationMachine = useConfigurationMachineService();
+
   const customGoBack = () => configurationMachine.send({ type: "BACK" });
 
+  const [modalSnapPoint, setModalSnapPoint] = React.useState<number>(100);
+
   useNavigationSwipeBackListener(() => {
     configurationMachine.send({ type: "BACK", skipNavigation: true });
   });
 
+  const handleOnLayout = (e: LayoutChangeEvent) => {
+    const { height } = e.nativeEvent.layout;
+    setModalSnapPoint(height + 200);
+  };
```

Comment:
I think adding this to `bottomSheet.tsx` would solve some future (and probably also past) layout/compatibilty problems. what do you think about it?

Reply:
I think that it's better to not change something used elsewhere because could lead to some unexpected behavior in the app.
But adding an "auto size" feature to `useIOBottomSheet` could be a nice improvement. Could be discussed for a future PR.